### DISCUSSION TITLE
Vitest Coverage UI

### DISCUSSION THREAD

**Discussion by ademyalcin27** [vitest-dev/vitest#6491] [Q&A] [answered]
There is column( green fields)  which is shows how many times is tested such as **2x, 3x**, how can I do visible on vitest? 

![image](https://github.com/user-attachments/assets/3a66a271-2cba-41fd-b6b4-866b9e03f9f2)

**Comment by AriPerkkio** [comment]
> how can I do visible on vitest

Sorry I'm not sure I understand the question. Could you be more specific? What's not visible?

**Reply by ademyalcin27** [reply]
The section I marked with a red mark below is not available on the current coverage UI screen. Here is information showing which line has been tested and how many times. This is available in the Jest infrastructure. But I didn't see in vitest UI

![367232886-3a66a271-2cba-41fd-b6b4-866b9e03f9f2](https://github.com/user-attachments/assets/2a329f30-3fb6-499e-8c1f-c6374aef647d)

**Reply by AriPerkkio** [reply, marked as answer]
It's visible on Vitest too, both on coverage HTML report and on Vitest UI. 

> This is available in the Jest infrastructure. But I didn't see in vitest UI

As far as I know, Jest doesn't have "UI" package. Are you opening the `coverage/index.html` from Jest in browser? You can do same with Vitest. Or just look at it on Vitest UI:

- https://stackblitz.com/~/edit/vitest-dev-vitest-djkxrz

![image](https://github.com/user-attachments/assets/fbe08fa8-7d0a-4e5e-ba2e-81b06158ceac)

**Reply by ademyalcin27** [reply]
I think I understand the problem I wasn't looking to coverage page. I was looking at this page. Thank you this helped me 

![Screenshot 2024-09-13 at 14 03 21](https://github.com/user-attachments/assets/aff74669-8f25-4e59-9a19-2307ab93a7ae)
